codespell_lib/data/dictionary.txt

```diff
@@ -17148,35 +17208,71 @@ deplying->deploying
 deplyment->deployment
 deplyments->deployments
 deplys->deploys
+depnd->depend
+depndance->dependence
+depndances->dependences
+depndancies->dependencies
+depndancy->dependency
 depndant->dependent
+depndantly->dependently
+depndants->dependents
+depnded->depended
+depndence->dependence
+depndences->dependences
+depndencies->dependencies
+depndency->dependency
+depndent->dependent
+depndently->dependently
+depndents->dependents
+depnding->depending
 depnds->depends
 deporarily->temporarily
 deposint->deposing
 depoy->deploy
 depoyed->deployed
 depoying->deploying
+depoyment->deployment
+depoyments->deployments
+depoys->deploys
```

Comment:
No occurrences of `depots` or `decoys`? Same for singular.

Reply:
I will add both.